### PR TITLE
[BUGFIX AND DEV] Added linting to package json, fixed colors and tab closing

### PR DESCRIPTION
**Pull request by laaledesiempre**
This are the changes:

- [x] Missing tab closing button (#11 )
- [x] Some colors didn't match (#7 )
- [x] eslint execution was not on package json (#10 )

Now, the thing is, i commented a lot of code on the js file because was not useful since now the button is fixed and use to broke all. but kept it. maybe we can delete it, or not, on future changes.

white colors now depends on the --white variable at `/styles.css`, so, i tested it on my screen but if you or someone finds that a different kind of white is better. now is easy to change, this change is to make it mode comfortable to read on any kind of screens, not eye burning white. also as now depends on a variable a future theme selector would be easier to make

now running eslint is as easy as exec `npm run lint`. this is useful if , in the future, we have multiple things to execute, or we change linter or whatever, it works fine rn. but let me know if somethings not working as intended

### CODE REVIEW

**Comment by laaledesiempre** [comment]
not used variable made eslint scream so now is commented.

**Comment by laaledesiempre** [comment]
forgor to add the new script on the readme.

**Comment by Dpeta** [comment]
Closing tabs works good 👍 also easier to press this way on mobile than the previous tiny x. Just leaving the old code commented out for now is probably fine.

Don't think we should replace white with a different color by default though. It's softer on the eyes but the green-ish background makes it look pretty different than what Pesterchum normally looks like, I'd rather leave smthin like that as a theme thing or an option.

**Comment by laaledesiempre** [comment]
Agree, was a mistake, now it "white". it should be this white i dont know why it was like "green". also if you dont like it, is easy to change now, i still have to make all colors responsive to variables for the future theme implementation. see if you like it and tell me.

File: README.md [hunk 1]
```diff
@@ -35,3 +35,11 @@ Run source through browserify + babel, equivalent to running bundle.sh:
...
 ```
 Then input the resulting hashes, same as the previous setup.
+
+# Contributing
+> [!NOTE]  
+> This section still WIP
+
+Before sending a pull request, if you modified the `/src/pesterchum.js` file, please run 
+`npm run lint`
+This will run eslint automatic linter and scream about errors, it needs to be flawless for the `CI/CD` building.
```
Comment (Dpeta) [inline README.md:45]:
```suggestion
This will run ESLint (``eslint --fix``) and automatically fix problems and enforce style guidelines. 
```

**Review by Dpeta — CHANGES_REQUESTED** [review]
3rg5e0fcpn8c12

**Comment by laaledesiempre** [comment]
okey changed are now on.

**Comment by Dpeta** [comment]
woo yeah woo yeah

**Pull request merged**